modules/logging-svc/package.json

```diff
@@ -30,7 +30,7 @@
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
     "test:unit": "jest --passWithNoTests --testMatch '**/test/unit/**/*.test.ts'",
-    "test:integration": "jest --passWithNoTests --testMatch '**/test/integration/**/*.test.ts'",
+    "test:integration": "jest --passWithNoTests --detectOpenHandles --testMatch '**/test/integration/**/*.test.ts'",
```

Comment:
Why use --detectOpenHandles only on test:integration?

Reply:
When I ran the integration test , it was failing because of some open handles . I just added those for testing. I will remove it.